src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,34 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
+    public static final DateTimeFormatter DATE_FORMAT = DateTimeFormatter.ofPattern("dd.MM.yyyy");
+
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        LocalDate dateFromDateFormat = LocalDate.parse(dateFrom, DATE_FORMAT);
+        LocalDate dateToDateFormat = LocalDate.parse(dateTo, DATE_FORMAT).plusDays(1);
```

Comment:
```suggestion
        LocalDate localDateFrom = LocalDate.parse(dateFrom, DATE_FORMAT);
        LocalDate LocalDateTo = LocalDate.parse(dateTo, DATE_FORMAT).plusDays(1);
```

why do you add one day to LocalDateTo?